src/librustc_traits/chalk/db.rs

```diff
@@ -445,24 +444,28 @@ impl<'tcx> chalk_solve::RustIrDatabase<RustInterner<'tcx>> for RustIrDatabase<'t
                 .tcx
                 .lang_items()
                 .sized_trait()
+                .require()
                 .map(|t| chalk_ir::TraitId(RustDefId::Trait(t)))
                 .unwrap(),
             CopyTrait => self
                 .tcx
                 .lang_items()
                 .copy_trait()
+                .require()
                 .map(|t| chalk_ir::TraitId(RustDefId::Trait(t)))
                 .unwrap(),
```

Comment:
This is the `require().unwrap()` pattern I was worried about BTW.

Reply:
Yeah I definitely agree - it’s just about trying to find a way to avoid having the whole compiler import the new type